pages/index.vue

```diff
@@ -1,13 +1,17 @@
 <template>
-  <UContainer class="space-y-8">
-    <UContainer>
-      <form @submit.prevent="uploadHandler" class="flex gap-2 justify-center">
-        <UInput type="file" multiple id="file" accept="image/*" />
-        <UButton type="submit" variant="outline" :loading="uploading">{{
-          $t("upload.fileUploader.uploadButton")
-        }}</UButton>
-      </form>
-    </UContainer>
+  <UContainer class="w-1/2 space-y-8">
+    <!--TODO: process to upload-->
+    <FilePond
+      allowMulti="true"
+      credits="false"
+      accept="image/*"
+      :label-idle="
+        $t('upload.filepond.labelIdle.part1') +
+        '<span class=\'filepond--label-action\'>' +
+        $t('upload.filepond.labelIdle.part2') +
+        '</span>'
+      "
+    />
     <UTabs
```

Comment:
```suggestion
    <ClientOnly>
      <FilePond
        allowMulti="true"
        credits="false"
        accept="image/*"
        :label-idle="
          $t('upload.filepond.labelIdle.part1') +
          '<span class=\'filepond--label-action\'>' +
          $t('upload.filepond.labelIdle.part2') +
          '</span>'
        "
      />
    </ClientOnly>
```
maybe not ssr compatible

Reply:
oh, some mistakes on choosing the area